modules/build_fcitx5.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+WPS_LIBRARY_DIR=$PWD/deb-package/opt/kingsoft/wps-office/office6
+
+sed -i 's/^CFG_INOTIFY=.*/CFG_INOTIFY=no/g' configure
+./configure -opensource -confirm-license -prefix $PWD
+# Build only the necessary part
+make -C src/tools/bootstrap
```

Comment:
Please set number of jobs for this make call and following, `-j$FLATPAK_BUILDER_N_JOBS`

Reply:
Thanks! I didn't know about this var :D.